tools/licenses/lib/main.dart

```diff
@@ -267,12 +195,10 @@ class _RepositoryReadmeIjgFile extends _RepositorySingleLicenseFile {
     r'unaltered; and any additions, deletions, or changes to the original files\n'
     r'must be clearly indicated in accompanying documentation\.\n'
     r'\(2\) If only executable code is distributed, then the accompanying\n'
-    r'documentation must state that "this software is based in part on the work of\n'
-    r'the Independent JPEG Group"\.\n'
+    r'documentation must state that "' '${_message.replaceAll(" ", "[ \n]+")}' r'"\.\n'
```

Comment:
Should we have a named constant for `"[ \n]+"` somewhere, so it's easier to understand in context?

Reply:
added a comment explaining what it did instead. (I couldn't come up with a good name.)